oneandone/resources_oneandone_block_storage.go

```diff
@@ -0,0 +1,167 @@
+package oneandone
+
+import (
+	"fmt"
+	"github.com/1and1/oneandone-cloudserver-sdk-go"
+	"github.com/hashicorp/terraform/helper/schema"
+	"strings"
+)
+
+func resourceOneandOneBlockStorage() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceOneandOneBlockStorageCreate,
+		Read:   resourceOneandOneBlockStorageRead,
+		Update: resourceOneandOneBlockStorageUpdate,
+		Delete: resourceOneandOneBlockStorageDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"size": {
+				Type:     schema.TypeInt,
+				Required: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"datacenter": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"server_id": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: suppressServerIdFunc,
```

Comment:
suppressServerIdFunc can be replaced with just true since the function doesn't return anything else.

Reply:
Although in this case it does not have a complex logic - the value for `DiffSuppressFunc` has to be of type `SchemaDiffSuppressFunc`,